paywall/src/unlockTypes.ts

```diff
@@ -91,6 +91,11 @@ export interface Lock {
   keyPrice: string
   expirationDuration: number
   key: Key
+  asOf?: number
```

Comment:
Not something that has to be part of this PR, but in the future we may want to have a lock be any of a union type of different lock states, so we can encode when we expect the properties that are optional here to be set

Reply:
I don't follow (typescript novice), looking forward to picking your brain on this in the near future!